fix parsing of "df" output

Previously '^A' was used as a pattern reference. This seemed to have no effect.
Changed the reference to '\1'.
--- /dev/null
+++ b/packages/admin/muninlite/patches/300-fix-df-parsing.patch
@@ -0,0 +1,20 @@
+--- a/plugins/df
++++ b/plugins/df
+@@ -15,7 +15,7 @@
+ graph_vlabel %
+ graph_category disk
+ graph_info This graph shows disk usage on the machine."
+-  for PART in $(df -P | grep '^/' | sed '/\/[a-z0-9]*$/!d;s/.* \([a-z0-9\/]\{1,\}\)$//g')
++  for PART in $(df -P | grep '^/' | sed '/\/[a-z0-9]*$/!d;s/.* \([a-z0-9\/]\{1,\}\)$/\1/g')
+   do
+     PINFO=$(df -P $PART | tail -1);
+     PNAME=$(echo $PINFO | cut -d\  -f1 | sed 's/\//_/g')
+@@ -26,7 +26,7 @@
+   done
+ }
+ fetch_df() {
+-  for PART in $(df -P | grep '^/' | sed '/\/[a-z0-9]*$/!d;s/.* \([a-z0-9\/]\{1,\}\)$//g')
++  for PART in $(df -P | grep '^/' | sed '/\/[a-z0-9]*$/!d;s/.* \([a-z0-9\/]\{1,\}\)$/\1/g')
+   do
+     PINFO=$(df -P $PART | tail -1);
+     PNAME=$(echo $PINFO | cut -d\  -f1 | sed 's/[\/.-]/_/g')
